src/ioctls/system.rs

```diff
@@ -598,6 +626,18 @@ mod tests {
         assert!(kvm.get_nr_memslots() >= 32);
     }
 
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    #[test]
+    fn test_kvm_get_disable_exits() {
+        let kvm = Kvm::new().unwrap();
+        let disable_exits = kvm.get_disable_exits();
+
+        if disable_exits != 0 {
+            assert!(disable_exits & KVM_X86_DISABLE_EXITS_HLT != 0);
+            assert!(disable_exits & KVM_X86_DISABLE_EXITS_PAUSE != 0);
```

Comment:
Running this test on my laptop fails, should we check here for some capability before doing the asserts? Maybe this way we can also have a test case for `get_disable_exits` returning 0.